Makefile

```diff
@@ -66,11 +66,11 @@ endif
 
 $(NODE_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Release V=$(V)
-	ln -fs out/Release/$(NODE_EXE) $@
+	if [ ! -r $(NODE_EXE) ]; then ln -fs out/Release/$(NODE_EXE) $@; fi
```

Comment:
Wouldn't -L (symbolic link) make more sense?

Can you add a comment here and below explaining why the check is necessary?  It's going to look superfluous to the casual reader.